scripts/build_char2array.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import pickle
+from os import listdir
+import numpy as np
+from src.utils.common import text2image
+from fontTools.ttLib import TTFont
+from loguru import logger
+from tqdm import tqdm
+import ftfy
+
+
+def text2numpy(text: str, font: str, font_size: int = 16):
+    image = text2image(text, font=font, font_size=font_size)
+    image_bytes = np.array(image)
+    image_bytes = 255 - image_bytes
+    return image_bytes
+
+
+# Check whether font supports symbol
+def has_glyph(font: TTFont, glyph: str) -> bool:
+    assert len(glyph) == 1, glyph
+    for table in font["cmap"].tables:
+        if ord(glyph) in table.cmap.keys():
+            return True
+    return False
+
+
+# Find correct font that supports symbol or returns None
+def pick_font(symbol: str, fonts: list) -> str | None:
+    for font in fonts:
+        if has_glyph(TTFont(font), symbol):
+            return font
+    return None
+
+
+def build_char2array(
+    save_to: str,
+    fonts_directory: str,
+    chars_path: str,
+    font_size: int = 16,
+    fill_value_unknown: int = 255,
+    width_unknown: int = None,
+    ignore_invalid_encoding: bool = False,
+):
+    assert fill_value_unknown in range(256), "Value of filling must be int number in [0,255]"
```

Comment:
I think you can define `fill_value_unknown` as a global constant. I don't think that we would change this value often.